.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+
+node_js:
+  - "0.10"
+  - "0.12"
```

Comment:
IIRC we talked about dropping 0.12, but let's track that across all our repos in a separate bug